### PR TITLE
Gui work

### PR DESCRIPTION
**Pull request by charleskhenry**
Looking to merge with master.  Only contains main gui.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,7 @@
 .DS_Store
+<<<<<<< HEAD
```
Comment (Shakiem) [inline .gitignore]: You should resolve these conflicts... take out the <<<<<< HEAD etc

File: .project [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Shakiem) [inline .project]: You should add all IDE files to the .gitignore file, they do not need to be apart of the repo.

**Comment by charleskhenry** [comment]
Changes had been made

File: .gitignore [hunk 3]
```diff
@@ -1,7 +0,0 @@
-.DS_Store
-<<<<<<< HEAD
-/bin/
-=======
-.idea
-out
->>>>>>> f831f33f606050dcbcd9671e942be2c5c19851f2
```
Comment (Shakiem) [inline .gitignore]: You still want to tell git to ignore the files, so you can't remove everything from .gitignore. It should look like this:

.DS_Store
/bin/
.idea
out
.project
.classpath

**Comment by charleskhenry** [comment]
Third time a charm?

**Pull request merged**